cmd/minikube/cmd/logs.go

```diff
@@ -66,7 +67,7 @@ var logsCmd = &cobra.Command{
 		if err != nil {
 			exit.WithError("command runner", err)
 		}
-		bs, err := getClusterBootstrapper(api, viper.GetString(cmdcfg.Bootstrapper))
+		bs, err := node.Bootstrapper(api, viper.GetString(cmdcfg.Bootstrapper))
```

Comment:
This placement feels a bit weird: the bootstrapper is a cluster-property rather than a node property.